src/files/index.spec.js

```diff
@@ -0,0 +1,121 @@
+import Files from './index';
+import * as objectUtils from '../utils/objects';
+
+describe('Files', function() {
+  let baseRequest;
+  let baseSdk;
+  let expectedHost;
+
+  beforeEach(function() {
+    this.sandbox = sandbox.create();
+
+    baseRequest = {
+      delete: this.sandbox.stub().resolves(),
+      get: this.sandbox.stub().resolves(),
+      post: this.sandbox.stub().resolves(),
+      put: this.sandbox.stub().resolves()
+    };
+
+    baseSdk = {
+      config: {
+        audiences: {
+          files: fixture.build('audience')
+        }
+      }
+    };
+    expectedHost = faker.internet.url();
+  });
+
+  afterEach(function() {
+    this.sandbox.restore();
+  });
+
+  describe('constructor', function() {
+    let files;
+
+    beforeEach(function() {
+      files = new Files(baseSdk, baseRequest);
+    });
+
+    it('sets a base url for the class instance', function() {
+      expect(files._baseUrl).to.equal(
+        `${baseSdk.config.audiences.files.host}/v1`
+      );
+    });
+
+    it('appends the supplied request module to the class instance', function() {
+      expect(files._request).to.deep.equal(baseRequest);
+    });
+
+    it('appends the supplied sdk to the class instance', function() {
+      expect(files._sdk).to.deep.equal(baseSdk);
+    });
+  });
+
+  describe('get', function() {
+    context('the file ID is provided', function() {
+      let fileFromServerAfterFormat;
+      let fileFromServerBeforeFormat;
+      let expectedFileId;
+      let promise;
+      let request;
+      let toCamelCase;
+
+      beforeEach(function() {
+        expectedFileId = faker.random.uuid();
+        fileFromServerAfterFormat = fixture.build('file', {
+          id: expectedFileId
+        });
+
+        fileFromServerBeforeFormat = fixture.build(
+          'file',
+          { id: expectedFileId },
+          { fromServer: true }
+        );
```

Comment:
What do you think about doing this a little differently?

```
fileFromServerAfterFormat = fixture.build('file');
fileFromServerBeforeFormat = fixture.build('file', fileFromServerAfterFormat, { fromServer: true });
```

Reply:
Ah, I keep forgetting about that option.